### PR TITLE
Edge webextension api version updates

### PR DESCRIPTION
**Pull request by mattwojo**

### CODE REVIEW

File: webextensions/api/storage.json [hunk 1]
```diff
@@ -75,7 +75,10 @@
                   "version_added": true
                 },
                 "edge": {
-                  "version_added": true
+                  "version_added": "14",
+                  "notes": [
++                  "storage.local data is persisted in a different format than Chrome, causing a different value to be returned when calling storage.local.getBytesInUse. Ex: storage.local.set({ "k": { "s": "âas" } } returns 13 in Chrome and 50 in Microsoft Edge.",
++                ]
```
Comment (ExE-Boss) [inline webextensions/api/storage.json]: The `+`es at the beginning of the lines are causing JSON parsing errors.

File: webextensions/api/tabs.json [hunk 2]
```diff
@@ -1881,7 +1881,10 @@
                 "version_added": true
               },
               "edge": {
-                "version_added": true
+                "version_added": "14",
+                "notes": [
++                  "After uninstall/reinstall, the URL is not received until Microsoft Edge is restarted."
++                ]
```
Comment (ExE-Boss) [inline webextensions/api/tabs.json]: ditto

File: webextensions/api/tabs.json [hunk 3]
```diff
@@ -2797,7 +2801,10 @@
                 "version_added": true
               },
               "edge": {
-                "version_added": true
+                "version_added": "14",
+                "notes": [
++                  "`pinned` and `muted` are not yet supported. `highlighted` and `selected` are not supported."
++                ]
```
Comment (ExE-Boss) [inline webextensions/api/tabs.json]: ditto

File: webextensions/api/webRequest.json [hunk 4]
```diff
@@ -54,7 +54,13 @@
...
               },
               "edge": {
-                "version_added": true
+                "version_added": "14",
+                "notes": [
++                  "`webRequest` not supported for synchronous XmlHttpRequests.",
++                  "Network requests from extensions, such as options, background or popup pages, are not supported.",
+                   "Network requests from <object> and <embed> elements are not supported.",
+                   "Headers cannot be modified for cached requests."
++                ]
```
Comment (ExE-Boss) [inline webextensions/api/webRequest.json]: ditto

File: webextensions/api/storage.json [hunk 5]
```diff
@@ -77,8 +77,8 @@
                 "edge": {
                   "version_added": "14",
                   "notes": [
-+                  "storage.local data is persisted in a different format than Chrome, causing a different value to be returned when calling storage.local.getBytesInUse. Ex: storage.local.set({ "k": { "s": "âas" } } returns 13 in Chrome and 50 in Microsoft Edge.",
-+                ]
+                  "storage.local data is persisted in a different format than Chrome, causing a different value to be returned when calling storage.local.getBytesInUse."
+                ]
```
Comment (ExE-Boss) [inline webextensions/api/storage.json]: These lines have wrong indentation.

File: webextensions/api/webRequest.json [hunk 6]
```diff
@@ -56,11 +56,11 @@
               "edge": {
                 "version_added": "14",
                 "notes": [
-+                  "`webRequest` not supported for synchronous XmlHttpRequests.",
-+                  "Network requests from extensions, such as options, background or popup pages, are not supported.",
+                  "`webRequest` not supported for synchronous XmlHttpRequests.",
+                  "Network requests from extensions, such as options, background or popup pages, are not supported.",
                    "Network requests from <object> and <embed> elements are not supported.",
                    "Headers cannot be modified for cached requests."
```
Comment (ExE-Boss) [inline webextensions/api/webRequest.json]: ditto

File: webextensions/api/storage.json [hunk 7]
```diff
@@ -77,7 +77,7 @@
                 "edge": {
                   "version_added": "14",
                   "notes": [
-                  "storage.local data is persisted in a different format than Chrome, causing a different value to be returned when calling storage.local.getBytesInUse."
+                  "<code>storage.local</code> data is persisted in a different format than Chrome, causing a different value to be returned when calling <code>storage.local.getBytesInUse</code>."
                 ]
```
Comment (ExE-Boss) [inline webextensions/api/storage.json]: These lines still have the wrong indentation.

File: webextensions/api/webRequest.json [hunk 8]
```diff
@@ -1541,9 +1544,10 @@
               },
               "edge": {
                 "notes": [
-                  "Asynchronous event listeners are not supported."
+                  "Asynchronous event listeners are not supported.",
+                  "`requestBody` is not supported."
```
Comment (wbamberg) [inline webextensions/api/webRequest.json]: Should use `<code>requestBody</code>`, or just plaintext. Markdown is not supported in notes (yes I know lots of existing notes use it :-/

**Comment by ExE-Boss** [comment]
This could probably be merged with my effort in #1686.

**Comment by Elchi3** [comment]
This needs a rebase now that https://github.com/mdn/browser-compat-data/pull/1686 is merged.

**Comment by Elchi3** [comment]
No activity here, so I'm closing this now. Please re-open if you want to come back to it.

**Pull request closed without merging**